src/screens/FeedScreen/Content.js

```diff
@@ -11,6 +11,7 @@ import ImageLayouter from './elements/ImageLayouter';
 import TopicsChip from '../../components/TopicsChip/TopicsChip';
 import {COLORS} from '../../utils/theme';
 import {POST_TYPE_LINK, POST_TYPE_POLL, POST_TYPE_STANDARD} from '../../utils/constants';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `colors` has been added, but the old import statement for `COLORS` from `'../../utils/theme'` is still present. If `COLORS` is no longer used in this file, it should be removed to avoid confusion and unnecessary imports.
```diff
- import {COLORS} from '../../utils/theme';
+ // Removed unused import
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->